docs/systemd.md

```diff
@@ -0,0 +1,95 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+# Systemd units with Tezos binaries
+
+<a name="ubuntu-and-fedora"></a>
+## Systemd units on Ubuntu and Fedora
+
+`tezos-node`, `tezos-accuser-<proto>`, `tezos-baker-<proto>`,
+`tezos-endorser-<proto>`, and `tezos-signer` packages have systemd files included to the
+Ubuntu and Fedora packages.
+
+Once you've installed the packages with systemd unit, you can run the service
+with the binary from the package using the following command:
```

Comment:
```suggestion
Once you've installed one of these packages, you can run the service
using the following command:
```
Seems a bit redundant otherwise